lib/src/core/models/chat/sub_models/choices/sub_models/message.dart

```diff
@@ -51,6 +63,7 @@ final class OpenAIChatCompletionChoiceMessageModel {
 
     return other is OpenAIChatCompletionChoiceMessageModel &&
         other.role == role &&
-        other.content == content;
+        other.content == content &&
+        other.functionCall == functionCall;
```

Comment:
I guess this will have no effect currently, since the `FunctionCallResponse` does not `override` the `==` operator.